oapen-engine/src/model/ngrams.py

```diff
@@ -1,14 +1,17 @@
 import string
```

Comment:
This is the file that is not formatted properly and is breaking the CI job

Reply:
It's flake8 throwing errors and possibly black. I can try to fix the hooks for this or get rid of them today. Besides that, this PR is done